base/commands/update_check.go

```diff
@@ -0,0 +1,98 @@
+package commands
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/hazelcast/hazelcast-commandline-client/clc/paths"
+	"github.com/hazelcast/hazelcast-commandline-client/clc/store"
+	"github.com/hazelcast/hazelcast-commandline-client/internal"
+	. "github.com/hazelcast/hazelcast-commandline-client/internal/check"
+	"github.com/hazelcast/hazelcast-commandline-client/internal/plug"
+)
+
+const skipUpdateCheck = "CLC_SKIP_UPDATE_CHECK"
```

Comment:
Could you add this environment variable to the environment variable reference in the docs?